templates/java/build-alt.gradle.mustache

```diff
@@ -11,7 +11,7 @@ apply plugin: 'signing'
 apply plugin: 'com.google.protobuf'
 
 description = 'GRPC library for service {{api.name}}-{{api.version}}'
-group = "io.gapi"
+group = "com.google.api"
```

Comment:
Apiary API uses "com.google.apis" as the group id. I feel this is very close to that one. Should we just use the same group ID?